examples/client/Locomotion/src/pages/Payments/credit-cards.tsx

```diff
@@ -31,19 +51,79 @@ export default ({
   return (
     <CardsListContainer>
       <View>
-        {usePayments.paymentMethods.map((paymentMethod : any) => (
-          <PaymentMethodsContainer>
-            <CreditCardsContainer>
-              <PaymentMethod {...paymentMethod} onPress={() => navigate(MAIN_ROUTES.CARD_DETAILS, { paymentMethod })} />
-            </CreditCardsContainer>
-          </PaymentMethodsContainer>
-        ))}
+        <PaymentMethodsContainer>
+          {defaultMethod && defaultMethod?.id !== cashPaymentMethod.id
```

Comment:
you don't need 'defaultMethod && defaultMethod?.id' when you have defaultMethod**?**.id